src/static/css/cf-theme-overrides.less

```diff
@@ -35,7 +35,7 @@
 @link-underline:                @pacific;
 @link-text-visited:             @teal;
 @link-underline-visited:        @teal;
-@link-text-hover:               @pacific-50;
```

Comment:
Design manual still has this at pacific-50. Has this changed?
http://cfpb.github.io/design-manual/ui-toolkit/links.html